Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -821,6 +827,150 @@ namespace AzToolsFramework
                         PrefabIdentifiers::PrefabInstancePropagationEndUpdaterIdentifier, actionIdentifier);
                 }
             }
+
+            // Revert overrides on Component
+            {
+                AZStd::string actionIdentifier = "o3de.action.prefabs.revertComponentOverrides";
+                AzToolsFramework::ActionProperties actionProperties;
+                actionProperties.m_name = "Revert overrides";
+                actionProperties.m_description = "Revert all overrides on this component.";
+                actionProperties.m_category = "Prefabs";
+
+                m_actionManagerInterface->RegisterAction(
+                    EditorIdentifiers::MainWindowActionContextIdentifier,
+                    actionIdentifier,
+                    actionProperties,
+                    []()
+                    {
+                        if (auto prefabOverridePublicInterface =
+                                AZ::Interface<AzToolsFramework::Prefab::PrefabOverridePublicInterface>::Get())
+                        {
+                            EntityPropertyEditorRequestBus::Broadcast(
+                                &EntityPropertyEditorRequestBus::Events::VisitComponentEditors,
+                                [prefabOverridePublicInterface](const AzToolsFramework::ComponentEditor* componentEditor)
+                                {
+                                    if (componentEditor->IsSelected())
+                                    {
+                                        auto components = componentEditor->GetComponents();
+
+                                        if (components.size() > 1)
+                                        {
+                                            // We do not support multi-editing with overrides.
```

Comment:
Should this be an `AZ_Warning`? (unless we don't warn on Editor actions)

Reply:
There's previous checks that should prevent this menu item from even being accessible to the user when multiple components are selected, so this check is just there to catch strays.